pkg/kubelet/dockershim/network/cni/cni_test.go

```diff
@@ -78,8 +82,8 @@ echo "%@" >> {{.OutputEnv}}
 export $(echo ${CNI_ARGS} | sed 's/;/ /g') &> /dev/null
 mkdir -p {{.OutputDir}} &> /dev/null
 echo -n "$CNI_COMMAND $CNI_NETNS $K8S_POD_NAMESPACE $K8S_POD_NAME $K8S_POD_INFRA_CONTAINER_ID" >& {{.OutputFile}}
-echo -n "{ \"ip4\": { \"ip\": \"10.1.0.23/24\" } }"
-`
+echo -n "{ \"cniVersion\": \"{{.CNIVersion}}\", \"ip4\": { \"ip\": \"{{.PodIP}}/24\" } }"`
```

Comment:
are the podIP changes actually needed here or is this a leftover part of #69821?

Reply:
yes, this var was used from previous commit, I'll remove it for this series .

Reply:
no, Dan changed the way of formatting, now it looks graceful.